tests/init/neg/local-warm4.check

```diff
@@ -1,18 +1,20 @@
 -- Error: tests/init/neg/local-warm4.scala:18:20 -----------------------------------------------------------------------
 18 |                a = newA // error
    |                    ^^^^
-   |       Cannot prove that the value is fully initialized. May only assign fully initialized value. Calling trace:
-   |       -> val a = new A(5)	[ local-warm4.scala:26 ]
-   |                  ^^^^^^^^
-   |       -> class A(x: Int) extends Foo(x) {	[ local-warm4.scala:6 ]
-   |          ^
-   |       -> val b = new B(y)	[ local-warm4.scala:10 ]
-   |                  ^^^^^^^^
-   |       -> class B(x: Int) extends A(x) {	[ local-warm4.scala:13 ]
-   |          ^
-   |       -> class A(x: Int) extends Foo(x) {	[ local-warm4.scala:6 ]
-   |          ^
-   |       -> increment()	[ local-warm4.scala:9 ]
-   |          ^^^^^^^^^^^
-   |       -> updateA()	[ local-warm4.scala:21 ]
-   |          ^^^^^^^^^
+   |                    May only assign fully initialized value. Calling trace:
```

Comment:
Again, I think there are two issues here:
- Why is line 18 called from a constructor? (Maybe say from the constructor of which class, in this case localWarm, through a stack of several other calls.)
- Why is newA not considered fully initialized? (In this case, because it comes from `new A(y)`, and `y` is not fully initialized. Why is not fully initialized? Because the tree `new A(y)` executes before the initializer of `A.y` runs. Why before? Because the constructor of B, which is a subclass of class A that contains the initializer of y, calls increment (line 23), which calls updateA (line 21), which contains the tree `new A(y)` (line 17). )